src/components/UplaodingDisplay/components/FileItem/FileItem.js

```diff
@@ -0,0 +1,42 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+
+import Progress from '../../../Progress';
+import css from './FileItem.css';
+
+class FileItem extends PureComponent {
+  static propTypes = {
+    name: PropTypes.string.isRequired,
+    size: PropTypes.number.isRequired,
+    currentUploaded: PropTypes.number,
+  };
+
+  static defaultProps = {
+    currentUploaded: 0,
+  };
+
+  render() {
+    const {
+      name,
+      currentUploaded,
+      size,
+    } = this.props;
+
+    return (
+      <div className={css.fileItem}>
+        <span>{name}</span>
+        <Progress
+          message="Uploading"
```

Comment:
Please, check do we use this `message` as a message displayed to the user and if so - use `<FormattedMessage>` with translation